220320 DialogSystem

chldbstj4536 opened the pull request:
feat: 다이얼로그 타이핑애니메이션, 텍스트 효과(흔들림, 웨이브, 무지개)

chldbstj4536 committed:
220320 DialogSystem

feat: 다이얼로그 타이핑애니메이션, 텍스트 효과(흔들림, 웨이브, 무지개)

jessi68 commented:
하나의 종류로만 쓴다면 확실히 굳이 클래스로 만들 필요는 없겠네요. 이해됬습니다~.   아하 매 프래임마다 연산할 필요가 없다고 생각해서 그렇군요. 이해됬습니다.

jessi68 commented on Assets/YS/1_Script/CustomTMPEffect.cs:
```diff
@@ -0,0 +1,231 @@
+        isDoneTyping = false;
+
+        textComponent.ForceMeshUpdate();
+        StartCoroutine(TypingStart());
```
코드 리뷰 예시입니다. 이 댓글처럼 제가 리뷰를 달면 어떤 코드에 달았는지 보실 수 있습니다.

chldbstj4536 replied:
오 감사합니다! :D

chldbstj4536 committed:
220327 다이얼로그 기능 구현

feat: 다이얼로그 스크립트 구조체 구현, 게임매니저를 통해 다이얼로그 불러오기 구현, UI 슬라이드 등 UI 배치 및 기능 구현 (메뉴보기, 로그기능)
